src/main/resources/application.properties

```diff
@@ -33,4 +33,5 @@ spring.data.cassandra.sb.username=cassandra
 spring.data.cassandra.sb.password=cassandra
 
 sunbird.course.service.host=https://staging.open-sunbird.org/api/course/
```

Comment:
The value should contain - http://lms-service:9000

Reply:
Pushed the changes